client/src/components/heroSection/heroSection.tsx

```diff
@@ -0,0 +1,51 @@
+import CssBaseline from '@mui/material/CssBaseline';
+import Container from '@mui/material/Container';
+import { Box, Typography } from '@mui/material';
+
+import { ThemeProvider } from '@emotion/react';
+import useStyles, { theme } from './heroStyles';
+
+function HeroSection() {
+  const classes = useStyles();
+  return (
+
+    <section className={classes.heroSection}>
+      <Box
+        className={classes.heroImage}
```

Comment:
I'm really sorry to suggest using `makeStyles`, It seems deprecated in the current version SO may convert to `sx` instead!